File: Lektioner/Lektion/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Log.d("TAG", "onCreate: Hello World");

        TextView helloThere = (TextView) findViewById(R.id.hello);


        Button button = (Button) findViewById(R.id.button);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                helloThere.setText("Hello world for real");

            }
        });


    }
}

File: Lektioner/Lektion6/app/src/main/java/com/example/lektion6/MainActivity.kt
package com.example.lektion6

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.TextView
import kotlin.math.log

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //Variables
        val STATIC: Int = 1;
        var change: Int = 0;

        var testString: String? = null;

        Log.d("test", STATIC.toString() + change)
        Log.d("test", "$STATIC + $change")

        if(change<1){
            Log.d("test", "$change")
            change++
        }

        var ifAsExpress:String = if(change<2){"if"}else{"else"}
        Log.d("test",ifAsExpress)

        logValue();

        val sampleClass = sampleClass()
        sampleClass.HELLO_STRING

        sampleObject.dogName="fido"
        Log.d("test",sampleObject.dogName)

        var helloText:TextView = findViewById(R.id.helloText)
        helloText.text="Hello from Kotlin Break.nu"

        





    }

    private fun logValue() {
        Log.d("test", "logValue"+makeLog())
    }
    private fun makeLog():String {

        return "makeLog"
    }
}